src/main_thread/api/debug/modules/segment_buffer_content.ts

```diff
@@ -75,13 +79,29 @@ export default function createSegmentSinkGraph(
 
   function updateBufferMetrics() {
     const showAllInfo = isExtendedMode(parentElt);
-    const inventory = bufferMetrics?.segmentSinks[bufferType].segmentInventory;
+    const metricsForType = bufferMetrics?.segmentSinks[bufferType];
+    const inventory = metricsForType?.segmentInventory;
     if (bufferMetrics === null || inventory === undefined) {
       bufferGraphWrapper.style.display = "none";
+      bufferSizeElt.innerHTML = "";
       currentRangeRepInfoElt.innerHTML = "";
       loadingRangeRepInfoElt.innerHTML = "";
     } else {
       bufferGraphWrapper.style.display = "block";
+      if (metricsForType?.sizeEstimate !== undefined) {
+        const sizeEstimate = metricsForType.sizeEstimate;
+        let sizeStr: string;
+        if (sizeEstimate > 2e6) {
+          sizeStr = (sizeEstimate / 1e6).toFixed(3) + "MB";
+        } else if (sizeEstimate > 2e3) {
+          sizeStr = (sizeEstimate / 1e3).toFixed(3) + "kB";
```

Comment:
`"kB"` has a different casing than `"MB"`

Reply:
Indeed, astute observation (https://en.wikipedia.org/wiki/International_System_of_Units#Prefixes)

Reply:
oh, never noticed that